packages/react/src/themes/teams/components/Icon/svg/icons/index.ts

```diff
@@ -207,6 +209,8 @@ export default {
   'canvas-add-page': canvasAddPage,
   chat,
   'chevron-down': chevronDown,
+  'chevron-start': chevronStart,
```

Comment:
Not sure if we should export these icons. Can you please just export the stardust-* icons